doc/api/worker_threads.md

```diff
@@ -4,21 +4,20 @@
 
 > Stability: 1 - Experimental
 
-The `worker` module provides a way to create multiple environments running
-on independent threads, and to create message channels between them. It
-can be accessed using the `--experimental-worker` flag and:
+The `worker_threads` module enables the use of threads with message channels
+between them. To access it, use the `--experimental-worker` flag and:
 
 ```js
 const worker = require('worker_threads');
 ```
 
-Workers are useful for performing CPU-intensive JavaScript operations; do not
-use them for I/O, since Node.js’s built-in mechanisms for performing operations
-asynchronously already treat it more efficiently than Worker threads can.
+Workers (threads) are useful for performing CPU-intensive JavaScript operations.
+Do not use them for I/O-intensive work. Node.js’s built-in asynchronous I/O
```

Comment:
I think this should be clarified. Current wording sounds too prohibitive, like there should be no I/O from workers period - similar to how Web Workers cannot access DOM.

Maybe - "While it is possible to offload I/O operations to the worker threads, often it will not improve performance compared to using asynchronous I/O on the main thread."

Reply:
The original intention was to actively discourage I/O from Workers, yes, because people often come to Node.js from other programming languages and assume that threading is the best way of doing parallel I/O.

I think sounding prohibitive is okay, but yeah, maybe the previous wording was better because it explained *why* offloading I/O is not a good idea, and the text in this PR does not explain that?

Reply:
Instead of "Do not use them", I could go with something like "They will not help much with I/O-intensive work. Node.js’s built-in asynchronous I/O is better suited for efficient I/O handling." I need to edit that but something roughly along those lines?